<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label00">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label01">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label02">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label03">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label04">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label05">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label06">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label07">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label08">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>100</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line01">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>40</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1 Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line02">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2 Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="turn1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="turn2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rstScr">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Score</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
